manifests/g/GottCode/FocusWriter/1.8.3/GottCode.FocusWriter.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: GottCode.FocusWriter
+PackageVersion: 1.8.3
+FileExtensions:
+- docx
+- fodt
+- odt
+- rtf
+- txt
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Application only works on x64 OS.